Sources/SwiftParserDiagnostics/ParseDiagnosticsGenerator.swift

```diff
@@ -452,6 +452,18 @@ public class ParseDiagnosticsGenerator: SyntaxAnyVisitor {
         message: { _ in .joinConditionsUsingComma },
         moveFixIt: { ReplaceTokensFixIt(replaceTokens: $0, replacement: trailingComma) }
       )
+    } else if let unexpected = (node.condition.as(AvailabilityConditionSyntax.self)?.unexpectedBetweenAvailabilitySpecAndRightParen) {
```

Comment:
Not sure if you need the left and right paren around 
```suggestion
    } else if let unexpected = node.condition.as(AvailabilityConditionSyntax.self)?.unexpectedBetweenAvailabilitySpecAndRightParen {
```

Reply:
Ty